### PR TITLE
Add editorconfig and remove idea folder

### PR DESCRIPTION
**Pull request by mattmook**

### CODE REVIEW

File: .idea/copyright/Orbit.xml [hunk 1]
```diff
@@ -1,7 +0,0 @@
-<component name="CopyrightManager">
```
Comment (Rosomack) [inline .idea/copyright/Orbit.xml:1]: Worth keeping the copyright file?

**Review by Rosomack — CHANGES_REQUESTED** [review]
Can we add these files to git ignore if they're not already there?

File: .idea/copyright/Orbit.xml (hunk 1 above)
Comment (mattmook) [reply .idea/copyright/Orbit.xml:1]: perhaps, although part of me just wants rid of the whole folder 🤣

Comment (Rosomack) [reply .idea/copyright/Orbit.xml:1]: Ah let's just nuke it, we've discussed there are caveats

**Review by Rosomack — APPROVED** [review]

**Pull request merged**